firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -249,6 +266,13 @@ class AppSettingsTableViewController: SettingsTableViewController,
             privacySettings.append(AutofillCreditCardSettings(settings: self, settingsDelegate: parentCoordinator))
         }
 
+        let autofillAddressStatus = featureFlags.isFeatureEnabled(.addressAutofill, checking: .buildOnly)
+        if autofillAddressStatus {
+            privacySettings.append(AddressAutofillSetting(theme: themeManager.currentTheme,
+                                                          profile: profile,
+                                                          settingsDelegate: parentCoordinator))
+        }
```

Comment:
@cyndichin so this in theory should take care of feature flag for v124

cc @jnrahme